### PR TITLE
Localization

### LINKED ISSUE #51: Fix volume being too loud at high values
I suggest adding a logarithmical scale when parsing user input in `volume()` command
BTW, we already have `import math`, might as well use it a bit more lol

### PR DESCRIPTION
**Pull request by duha54rus**
Adds a Russian translation to every message in the bot and its modules
Also fixes #51 (since a certain someone was really against pushing it to master...)

TODO before merging:
- [x] Add a feature to change the language of bot messages on startup

### CODE REVIEW

File: cogs/music.py [hunk 1]
```diff
@@ -182,77 +185,77 @@ async def download(self, ctx, url):
...
+    @commands.command(brief = _('Removes a song selected from the list'))
     async def remove(self, ctx, number = 0):
         if (1 <= int(number) <= len(self._songlist)):
             song = self._songlist.pop(int(number) - 1)
             try:
                 os.remove(self.music_path + song)
-                await self.boxed_print(ctx, f'Song {song[:-5]} has been deleted')
+                await self.boxed_print(ctx, self._('Song {} has been deleted').format(song[:-5]))
             except PermissionError:
                 await self.boxed_print(ctx, 'Unable to delete song file, probably it is being playing now.')
```
Comment (piunov1998) [inline cogs/music.py:199]: Need to translate this one

File: cogs/music.py [hunk 2]
```diff
@@ -182,77 +185,77 @@ async def download(self, ctx, url):
...
             except:
                 await self.boxed_print(ctx, 'Unable to delete song file')
```
Comment (piunov1998) [inline cogs/music.py:201]: and this one

**Review by piunov1998 — CHANGES_REQUESTED** [review]
It would be better to 

- [x] merge new `skip` command into `localization` branch, 
- [ ] translate it 
- [ ] and after all merge this one into `master`

**Comment by duha54rus** [comment]
Screw the 'on the fly' thing
On the start is the way lol

**Review by piunov1998 — CHANGES_REQUESTED** [review]
can u add missing translation?

**Review by piunov1998 — APPROVED** [review]
Add localization manual to readme file and merge

**Pull request merged**